dashboard/static/js/Roles.js

```diff
@@ -30,15 +30,75 @@ $(document).ready(function(){
         }
     });
     
+    function loadRolesTable(table, tableRows, columnAmt) {
+    var column_amt = columnAmt;
+    var inner_table = "";
+    var id, name, description;
+    inner_table += "<tbody>"
+
+    for (var row = 0; row < tableRows; row++){
+        id = json[row]["id"];
+        inner_table += "<tr>";
+        inner_table += "<td><input class='checkbox' type='checkbox' value='" + id + "' id='" + id + "' /></td>";
+        inner_table += "<td></td>";
+        inner_table += "<td></td>";
+        inner_table += "<td></td>";
+        inner_table += "</tr>";
+    };
+    inner_table += "</tbody>";
+    
+    $(table).append(inner_table);
+    
+    $('#deleteRoleButton').click(function(){
+        var rolesToDelete = [];
+        
+        $('.checkbox:checkbox:checked').each(function() {
+            rolesToDelete.push($(this).val());
+        });
+        
+        data = {
+            'id': rolesToDelete,
+        }
+        
+        console.log(rolesToDelete);
+        
+        $.ajax({
```

Comment:
If this ajax call is always going to be the same as the one above, then the code should be refactored to only have one.